app/Providers/AppServiceProvider.php

```diff
@@ -110,7 +111,29 @@ public function boot()
                 'endpoint' => $endpoint,
                 'scheme' => config('services.mailgun.scheme'),
             ]));
- 
+
+            return $this;
+        });
+
+        Mail::extend('brevo', function () {
+            return (new BrevoTransportFactory)->create(
+                new Dsn(
+                    'brevo+api',
+                    'default',
+                    config('services.brevo.key')
+                )
+            );
+        });
+        Mailer::macro('brevo_config', function (string $key) {
+            // @phpstan-ignore /** @phpstan-ignore-next-line **/
+            Mail::setSymfonyTransport((new BrevoTransportFactory)->create(
+                new Dsn(
+                    'brevo+api',
+                    'default',
+                    $key
+                )
+            ));
+
             return $this;
```

Comment:
@turbo124 please check exspecialy these lines, as I dont know if the implementation is correct.